brayns/io/Loader.h

```diff
@@ -81,8 +81,10 @@ class AbstractLoader
      * @brief Query the loader if it can load the given file. Compares the file
      * extension to the supported extensions (removing the dot, if any, and
      * making them lowercase). Can be overriden to perform mor explicit checks
+     * @param const std::string& fileName Path to the file to queried
+     * @param const std::string& extension Extracted file extension from fileName
      */
-    virtual bool isSupported(const std::string &filename, const std::string &extension) const
+    virtual bool isSupported(const std::string &, const std::string &extension) const
```

Comment:
Simple personal preference but I prefer naming the unused parameters and adding a (void)parameter in the function body so we still see what it is.

Reply:
Yes, that makes it more clear, Ill change that.